spec/features/guests/show_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.describe '/guests/:id', type: :feature do
+  describe "when I visit a guest show page" do
+    before do
+      @hotel1 = Hotel.create!(name: "Ritz-Carlton", location: "Paris")
+      @hotel2 = Hotel.create!(name: "St. Regis", location: "Geneva")
+
+      @room1 = Room.create!(rate: 300, suite: "Penthouse", hotel_id: @hotel1.id)
+      @room2 = Room.create!(rate: 400, suite: "Rooftop Bungalow", hotel_id: @hotel1.id)
+      @room3 = Room.create!(rate: 200, suite: "Loft", hotel_id: @hotel2.id)
+      @room4 = Room.create!(rate: 450, suite: "Honeymoon", hotel_id: @hotel2.id)
+
+      @guest1 = Guest.create!(name: "Letti")
+
+      GuestRoom.create!(room: @room1, guest: @guest1)
+      GuestRoom.create!(room: @room2, guest: @guest1)
+      GuestRoom.create!(room: @room3, guest: @guest1)
+
+      visit "/guests/#{@guest1.id}"
```

Comment:
Nice job with your object setup in your tests! 

One thing to be careful of in the future - if in any `it` blocks underneath this `before do` _alters_ any data or creates more relationships within the `it` blocks themselves, the page will have already been visited, and won't see that data updated. So, sometimes it is better to do the `visit...` line inside each `it` block, rather than in a `before :each`. Just something to keep in mind.